packages/tools/src/tools/displayTools/Labelmap/addLabelmapToElement.ts

```diff
@@ -36,15 +47,40 @@ async function addLabelmapToElement(
   const immediateRender = false;
   const suppressEvents = true;
 
-  if (isVolumeSegmentation(labelMapData, viewport)) {
+  if (viewport instanceof BaseVolumeViewport) {
```

Comment:
This bit feels like it should be in a helper method  'ensureVolumeHasVolumeId'   I think that would reduce the complexity of the method.